src/screens/Followings/index.js

```diff
@@ -28,5 +28,5 @@ Followings.propTypes = {
 export default Followings;
 
 const styles = StyleSheet.create({
-  container: {height: '100%', backgroundColor: colors.white}
+  container: {height: '100%', backgroundColor: COLORS.white}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `backgroundColor` property in the `container` style has been updated to use the new `COLORS` object. This is a good change as it ensures consistency in the use of color values throughout the application.
```diff
- container: {height: '100%', backgroundColor: colors.white}
+ container: {height: '100%', backgroundColor: COLORS.white}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->